core/core-awt/src/main/java/org/icepdf/core/pobjects/graphics/BlendComposite.java

```diff
@@ -101,6 +104,20 @@
         // sets the shadow colour of the decorator.
         disableBlendComposite = Defs.booleanProperty(
                 "org.icepdf.core.paint.disableBlendComposite", false);
+        try {
+            final Class xrSurfaceDataClass = Class.forName("sun.java2d.xr.XRSurfaceData$XRInternalSurfaceData");
+            final Constructor constructor = xrSurfaceDataClass.getDeclaredConstructor(Class.forName("sun.java2d.xr.XRBackend"), int.class);
+            final Object xrSurfaceData = constructor.newInstance(null, 0);
+            final Method getRaster = xrSurfaceDataClass.getMethod("getRaster", int.class, int.class, int.class, int.class);
+            getRaster.invoke(xrSurfaceData, 0, 0, 0, 0);
+        } catch (final InvocationTargetException e) {

```

Comment:
Curious as the special case handling here.  Wouldn't you want to disable blending on any error?

Reply:
I was wondering about that too. I guess it would be in case of a change in the API and the getRaster method would be renamed or have different parameters; in this case we would get an Exception but not an InternalError, but the method used to draw the composite could still work.       
Although it would be pretty unrealistic for this method to change, given that it's Sun internal API, it's always possible for this to happen.

Reply:
Alright, can you add some comments that at least give some context to the problem, test and catch structure.  .